demo/src/main/java/com/example/demo/Authorized/jwtAuthenticationFilter.java

```diff
@@ -0,0 +1,59 @@
+package com.example.demo.Authorized;
+
+import com.example.demo.Auth.User;
+import com.example.demo.Auth.UserRepository;
+import io.jsonwebtoken.ExpiredJwtException;
+import io.jsonwebtoken.MalformedJwtException;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+@Component
+public class jwtAuthenticationFilter extends OncePerRequestFilter {
```

Comment:
First letter of any class should be capital. rename it to JWTAuthenticationFilter

Reply:
same rule for other classes as well.